messages.properties

```diff
@@ -0,0 +1,95 @@
+isanteplus.title=iSantePlus Module
```

Comment:
this file is in the wrong place, what happened to your repo to have these files misplaced